examples/token/scripts/test.sh

```
@@ -0,0 +1,76 @@
+
+set -e
+set -x
+
+if [ "$1" = "localnet" ]; then
+  npx hardhat localnet --exit-on-error & sleep 10
+fi
+
+function balance() {
+  echo -e "\n🖼️  Balance"
+  echo "---------------------------------------------"
+  local ZETACHAIN=$(cast call "$CONTRACT_ZETACHAIN" "balanceOf(address)(uint256)" "$SENDER")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)